apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSourceIT.java

```diff
@@ -306,6 +306,56 @@ public void grpcCallWithIncorrectAuthToken() throws Exception {
             });
   }
 
+  /** Checks to see if a log message was NOT generated when a blank token is used. */
```

Comment:
When a null token is used rather than blank.  It'd be nice to refactor these into a common method since it appears the only difference between these two tests is `null` vs `""`.  Took me a while to determine the difference.

Reply:
Or maybe parameterize the test.